pkg/cmd/stack/translate.go

```diff
@@ -83,17 +85,41 @@ func translateStackEnvVars(s *model.Stack) error {
 	return nil
 }
 
-func translateServiceEnvFile(svc *model.Service, filename string) error {
+func translateServiceEnvFile(ctx context.Context, svc *model.Service, svcName, filename string, isOktetoNamespace bool) error {
 	var err error
 	filename, err = model.ExpandEnv(filename)
 	if err != nil {
 		return err
 	}
 
+	if filename == ".env" && isOktetoNamespace {
+		log.Information("[%s]: Detected '.env' file but will be skipped. You can set environment variables from https://cloud.okteto.com/#/settings/secrets.", svcName)
```

Comment:
why do we log this? since it's pretty common to have an .env file, this might look like we are telling users not to use that feature no?

Reply:
What happens if user has a local .env, but it's deploying on a namespace managed by okteto? maybe the .env generation is something that doesn't happen when deploying local, but only when inside the pipeline?

Reply:
@jLopezbarb let's do as we do for variable interpolation. Instead of omitting the content of `.env`, let's merge its content with the Okteto Secrets with more priority to the content of `.env`:
https://okteto.com/docs/reference/stacks#environment-variables

Reply:
And we can remove the log information message then